Update known_limitations.mdx (#2530)
meilisearch/documentation: learn/inner_workings/known_limitations.mdx

**Limitation:** Meilisearch can accommodate an arbitrary number of indexes as long as their size does not exceed 2TiB. When dealing with larger indexes, Meilisearch can accommodate up to 20 indexes as long as their combined size does not exceed the OS's virtual address space limit.

**Explanation:** While Meilisearch supports an arbitrary number of indexes under 2TiB, accessing hundreds of different databases in short periods of time might lead to decreased performance and should be avoided when possible.

## Facet Search limitation

> **Limitation:** The current limitation of retrievable facets is 100. When altering the facet amount the search will still only return 100.

Reviewer: It's usually better to be as concise as possible when talking about this type of limitation. My suggestion is to first state the context (when using facet search), then state the limit (maximum of 100 facets).

```suggestion
**Limitation:** When [searching for facet values](reference/api/facet_search), Meilisearch returns a maximum of 100 facets.
```
